src/README.md

```diff
@@ -0,0 +1,68 @@
+# Root source folder
```

Comment:
A bunch of monospaced strings in this file seem to have an extra space at the beginning (e.g. “`｀ Model｀`”).